src/App.js

```diff
@@ -1,22 +1,40 @@
 import React from 'react';
 import './App.css';
 import Settings from './routes/settings';
-import SurveyView from './routes/survey';
-import Search from './routes/search';
 import ViewSpace from './routes/view-space/the-think-tank';
-import Homebar from './components/homebar';
-
+import Search from './routes/search';
+import {
+  Survey,
```

Comment:
related. survey component is different from survey in routes. they're named horribly i know :(